homeassistant/components/music_assistant/manifest.json

```diff
@@ -8,6 +8,6 @@
   "iot_class": "local_push",
   "issue_tracker": "https://github.com/music-assistant/hass-music-assistant/issues",
   "loggers": ["music_assistant"],
-  "requirements": ["music-assistant-client==1.0.3"],
+  "requirements": ["music-assistant-client==1.0.5"],
```

Comment:
Please bump in a separate PR

Reply:
it was needed for the services as I moved the helpers there

Reply:
Yes, but we can merge that one first